ui/src/main/resources/PhenoTips/SimilarCases.xml

```diff
@@ -1684,137 +1807,108 @@
-      (% class="similar-cases-category similar-cases-local" %)(((
-      === $escapetool.xml($services.localization.render('phenotips.similarCases.subtitle')) ===
-      (% class="similarity-results-container-local" %)(((
-        (% class="similarity-results-container" id="similar-patients-container" %)((()))##
-        ## Pagination
-        $xwiki.ssfx.use('js/xwiki/table/livetable.css', true)##
-        $xwiki.ssfx.use('uicomponents/matchingNotification/contactDialog.css')##
-        $xwiki.jsfx.use('uicomponents/matchingNotification/matcherPaginator.js', true)##
-        $xwiki.jsfx.use('uicomponents/matchingNotification/matcherPageSizer.js', true)##
-        $xwiki.jsfx.use('uicomponents/matchingNotification/matcherContactDialog.js', true)##
-        $xwiki.jsfx.use('uicomponents/matchingNotification/notificationDialog.js', true)##
-        {{html clean=false}}&lt;span class="xwiki-livetable-pagination" id="pagination-similar-cases"&gt;&lt;span class="controlPagination"&gt;
-            &lt;a title="$escapetool.xml($services.localization.render('platform.livetable.paginationPagePrevTitle'))" class="prevPagination" id="pagination-prev" href="#"&gt;&lt;span class="hidden"&gt;$services.localization.render('platform.livetable.paginationPagePrevTitle')&lt;/span&gt;&lt;/a&gt;
-            &lt;a title="$escapetool.xml($services.localization.render('platform.livetable.paginationPageNextTitle'))" class="nextPagination" id="pagination-next" href="#"&gt;&lt;span class="hidden"&gt;$services.localization.render('platform.livetable.paginationPageNextTitle')&lt;/span&gt;&lt;/a&gt;
-          &lt;/span&gt;
-          &lt;span class="pagination"&gt;
-            &lt;span class="xwiki-livetable-pagination-text"&gt;$services.localization.render('platform.livetable.paginationPage')&lt;/span&gt;
-            &lt;span class="xwiki-livetable-pagination-content" id="pagination-content"&gt;&lt;/span&gt;
-          &lt;/span&gt;&lt;/span&gt;
-        {{/html}})))##
-      )))
-      ## Allow to plug in other data sources
-      #foreach ($e in $services.uix.getExtensions("org.phenotips.patientSheet.similarCases"))
-        (% class="similar-cases-category" %)(((
-        {{html clean=false}}$services.rendering.render($e.execute(), 'xhtml/1.0'){{/html}}
-        )))
-      #end
-    #end## can see similar patients
-  #end## suggestions enabled
-#end## action == view
-{{/velocity}}</content>
+      <cache>long</cache>
     </property>
     <property>
-      <extensionPointId>org.phenotips.patientSheet.after</extensionPointId>
+      <code>#template("colorThemeInit.vm")
+
+#document-head .title-tab:not(.active) {
+  background-color: $theme.buttonPrimaryBackgroundColor !important;
+  background-image: linear-gradient(to bottom,$theme.buttonPrimaryGradientColor 0,$theme.buttonPrimaryBackgroundColor 100%) !important;
+  font-weight: normal !important;
+  margin: 0 0 0 1px !important;
```

Comment:
There should be a margin right applied to all tabs instead of a margin left applied to non-active ones.
```suggestion
}
#document-head .title-tab {
  margin: 0 1px 0 0 !important;
```